File: hw/chicaiza/u2/hw19Inheritance/FarmSystemJava/src/main/java/ec/edu/espe/farmsystem/model/Pig.java
package ec.edu.espe.farmsystem.model;

import java.util.Date;

/**
 *
 * @author Daniel
 */
public class Pig extends FarmAnimal {

    @Override
    public String toString() {
        return "Pig{" + 
                "\n " +super.toString()+             
                "\n}";
    }

    
    public Pig(int id, String breed, Date bornOn, String gender, boolean isAbleToReproduce, float weight, Cage cage) {
        super(id, breed, bornOn, gender, isAbleToReproduce, weight, cage);
    }
    
     
}

File: hw/bonilla/u2/hw19Inheritance/FarmSystemPython/model/Pig.py
from model.FarmAnimal import FarmAnimal

class Pig(FarmAnimal):
    def __init__(self, id, breed, bornOn, gender, isAbleToReproduce, weight, cage):
        super().__init__(id, breed, bornOn, gender, isAbleToReproduce, weight, cage)

    def __str__(self):
        return f"""{{
        "farmAnimal": {super().__str__()}
        }}"""
